# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/*/openh323/pwlib.patch
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./tools/asnparser/asnparser.dsp	2002-10-08 14:41:52.000000000 +0200
+++ ./tools/asnparser/asnparser.dsp	2003-04-04 12:20:46.000000000 +0200
@@ -108,7 +108,7 @@
 InputName=asn_grammar
 
 BuildCmds= \
-	bison -t -v -d $(InputName).y \
+	bison -y -t -v -d $(InputName).y \
 	copy $(InputName)_tab.c $(InputName).cxx \
 	copy $(InputName)_tab.h $(InputName).h \
 	
@@ -127,7 +127,7 @@
 InputName=asn_grammar
 
 BuildCmds= \
-	bison -t -v -d $(InputName).y \
+	bison -y -t -v -d $(InputName).y \
 	copy $(InputName)_tab.c $(InputName).cxx \
 	copy $(InputName)_tab.h $(InputName).h \
 	
--- ./tools/asnparser/parser.patch	2003-04-04 12:42:10.000000000 +0200
+++ ./tools/asnparser/parser.patch	2003-04-04 12:40:15.000000000 +0200
@@ -0,0 +1,11 @@
+--- ./asn_grammar.cxx	2003-04-04 12:22:16.000000000 +0200
++++ ./asn_grammar.cxx	2003-04-04 12:39:11.000000000 +0200
+@@ -5045,7 +5045,7 @@
+ #if defined (__GNUC_MINOR__) && 2093 <= (__GNUC__ * 1000 + __GNUC_MINOR__)
+   __attribute__ ((__unused__))
+ #endif
+-
++	;
+ 
+   goto yyerrlab2;
+ 
--- ./tools/asnparser/Makefile~	2002-01-27 01:07:10.000000000 +0100
+++ ./tools/asnparser/Makefile	2003-04-04 12:45:12.000000000 +0200
@@ -55,8 +55,9 @@
 
 asn_grammar.cxx: asn_grammar.y
 	@if [ ! -d $(OBJDIR) ] ; then mkdir $(OBJDIR) ; fi
-	bison -t -v -d -o $* $<
+	bison -y -t -v -d -o $* $<
 	mv $* $*.cxx
+	patch -p 1 <parser.patch
 
 asn_lex.cxx: asn_lex.l
 	flex -t $< > $@
